docs/template-variables.rst

```diff
@@ -298,9 +301,9 @@ The archive navigation variables are available only if ``create_archive_navigati
 
 .. class:: table table-bordered table-striped
 
-==============================  ==============  ===============================================
+==============================  ==============  ========================================================================
 Name                            Type            Description
-==============================  ==============  ===============================================
```

Comment:
That must have gone missing when I tried to resolve conflicts with the github online editor. Re-added that.